### ISSUE TITLE
Is this launcher dead?

### ISSUE THREAD

**Issue by inson1** [sduduzog/slim-launcher#168]

**Comment by tsfoverlord** [comment]
Seems so. Its not on play store any more.

**Comment by sduduzog** [comment]
@tsfoverlord Yup, it was taken out due to a google play policy issue. The app is supposed to target Android 14 and higher.

Working on this though, it does mean rewriting from scratch in jetpack compose to mordenize the codebase.
